### PR TITLE
[Doc] Add Chinese doc for tutorials04_customized_models_md

### PR DESCRIPTION
**Pull request by MengzhangLI**
Add Chinese doc for tutorials04_customized_models_md

### CODE REVIEW

File: docs_zh-CN/tutorials/customize_models.md [hunk 1]
```diff
@@ -1 +1,230 @@
...
+
+        return my_optimizer
+
+```
+
+## 开发新的组件
+
+MMSegmentation 里主要有2种组件：
+
+- 骨架 (backbone): 通常是卷积网络的堆叠，来做特征提取，例如 ResNet, HRNet。
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_models.md:74]:
```suggestion
- 主干网络 (backbone): 通常是卷积网络的堆叠，来做特征提取，例如 ResNet, HRNet。
```

File: docs_zh-CN/tutorials/customize_models.md [hunk 2]
```diff
@@ -1 +1,230 @@
...
+- 头 (head): 用于语义分割图的解码的组件。
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_models.md:75]:
```suggestion
- 解码头 (head): 用于语义分割图的解码的组件。
```

File: docs_zh-CN/tutorials/customize_models.md [hunk 3]
```diff
@@ -1 +1,230 @@
...
+```
+
+使用者可以直接按照 PyTorch [文档教程](https://pytorch.org/docs/stable/optim.html?highlight=optim#module-torch.optim) 去设置参数。
+
+## 定制优化器的构造器 (optimizer constructor)
+
+对于优化，一些模型可能会有一些特别定义的参数，例如 批归一化 (BatchNorm) 层里面的权重衰减 (weight decay)。
+使用者可以通过定制优化器的构造器来微调这些细粒度的优化器参数。
+
+```
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_models.md:52]:
```suggestion
```python
```

File: docs_zh-CN/tutorials/customize_models.md [hunk 4]
```diff
@@ -1 +1,230 @@
...
+        type='MobileNet',
+        arg1=xxx,
+        arg2=xxx),
+    ...
+```
+
+### 增加新的头组件
+
+在 MMSegmentation 里面，对于所有的分割头，我们提供一个基础的 [BaseDecodeHead](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/decode_heads/decode_head.py) 。
+所有创新的解码头都应该由它生成。这里我们以 [PSPNet](https://arxiv.org/abs/1612.01105) 为例，
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_models.md:123]:
```suggestion
所有新建的解码头都应该由它生成。这里我们以 [PSPNet](https://arxiv.org/abs/1612.01105) 为例，
```

File: docs_zh-CN/tutorials/customize_models.md [hunk 5]
```diff
@@ -1 +1,230 @@
...
+    def __init__(self, pool_scales=(1, 2, 3, 6), **kwargs):
+        super(PSPHead, self).__init__(**kwargs)
+
+    def init_weights(self):
+
+    def forward(self, inputs):
+
+```
+
+接着，使用者需要在 `mmseg/models/decode_heads/__init__.py` 里面添加这个模块，这样对应的注册表 (registry) 可以查找并加载它们。
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_models.md:142]:
```suggestion
接着，使用者需要在 `mmseg/models/decode_heads/__init__.py` 里面添加这个模块，这样对应的注册器 (registry) 可以查找并加载它们。
```

**Comment by MengzhangLI** [comment]
Just pulled new version, please check it out.

Best,

**Review by Junjun2016 — APPROVED** [review]

**Pull request merged**